index.js

```diff
@@ -0,0 +1,72 @@
+import speaker from './module/data.js';
+
+const guestSpeacker = document.querySelector('#guest');
+// Mobile menue ------------
+const hamburger = document.querySelector('.hamburger');
+const mobileLink = document.querySelectorAll('.mobile-link');
+const mobileMenue = document.querySelector('.mobile-menu');
+function mobileToggleClassActive() {
+  hamburger.classList.toggle('active');
+  mobileMenue.classList.toggle('active');
+}
+
+hamburger.addEventListener('click', mobileToggleClassActive);
+mobileLink.forEach((element) => {
+  element.addEventListener('click', mobileToggleClassActive);
+});
+
+guestSpeacker.innerHTML = `
+<h2 class="feature-h2">Feature Speakers</h2>
+    <hr class="feature-hr">
+    <div class="feature-container">
+      <div class="speakers">
+        <div class="guestSpeaker">
+          <div class="guestSpeaker-img">
+            <img src="${speaker[0].image}" class="quest-img" alt="img">
+          </div>
+          <div class="content">
+            <h2 class="name">${speaker[0].name}</h2>
+            <p class="profession">${speaker[0].profession}
+            </p>
+
+            <p class="about-me">${speaker[0].aboutMe}</p>
+          </div>
+        </div>
+        <div class="guestSpeaker">
+          <div class="guestSpeaker-img">
+            <img src="${speaker[1].image}" class="quest-img" alt="img">
+          </div>
+          <div class="content">
+            <h2 class="name">${speaker[1].name}</h2>
+            <p class="profession">${speaker[1].profession}</p>
+
+            <p class="about-me">${speaker[1].aboutMe}</p>
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="guestSpeaker">
+          <div class="guestSpeaker-img">
+            <img src="${speaker[2].image}" class="quest-img" alt="img">
+          </div>
+          <div class="content">
+            <h2 class="name">${speaker[2].name}</h2>
+            <p class="profession">${speaker[2].profession}</p>
+
+            <p class="about-me">${speaker[2].aboutMe}</p>
+          </div>
+        </div>
+        <div class="guestSpeaker last">
+          <div class="guestSpeaker-img">
+            <img src="${speaker[3].image}" class="quest-img" alt="img">
+          </div>
+          <div class="content">
+            <h2 class="name">${speaker[3].name}</h2>
+            <p class="profession">${speaker[3].profession} </p>
+
+            <p class="about-me">${speaker[3].aboutMe}</p>
+          </div>
+        </div>
+      </div>
+    </div>
+`;
```

Comment:
- [ ] Kindly make sure the "Featured speakers" section is generated dynamically with JavaScript. Right now, you need to implement a function that iterates through the speakers list on window load and creates HTML elements with the data from each object.
```JavaScript
window.download = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    // or accumulate the generated elements in a variable and attach them all after it
  })
}
```

- [ ] Instead of repeating every time the HTML code for each card, I suggest you implement a feature so the dynamic content is attached to the static HTML by selecting the target parent element and adding the generated HTML with the DOM's `appendChild()` or `innerHTML()`. For example:
```JavaScript 
   const speakerContainer = document.getElementById("element-id")
  ...
  speakerContainer.appendChild(dynamicElements)
  // OR
  speakerContainer.innerHTML = dynamicElements
  // If used inside a loop the innerHTML method should use `+=` instead of just `=`
```